pages/public_cloud/containers_orchestration/managed_kubernetes/premium/guide.en-gb.md

```diff
@@ -0,0 +1,186 @@
+
+### API server admission plugins configuration
+
+The configuration of the [API server admission plugins](/pages/public_cloud/containers_orchestration/managed_kubernetes/apiserver-flags-configuration) is not available at the moment.
+
+### API Server IP restrictions
+
+To enable IP filtering on the API server, the IP of the gateway in the cluster's Openstack subnet should be specified.
+This allows worker nodes to communicate with the API server.
+
+Retreive the gateway IP of your cluster in the Manager, or the following command:
+
+```bash
+openstack router show ROUTER_ID -c external_gateway_info
+```
+
+### Security Policies
+
+Change the Security Policy after cluster creation is not supported yet.
+
+### Anti-affinity
+
+This feature allows worker nodes to be deployed on different hypervisors (physical servers), guaranteeing better fault tolerance. It is currently supported on the MKS Premium Plan (EU-WEST-PAR).
+
+We recommend using multiple Availability Zones (AZs) instead.
+
+### Ports
+
+The openstack security group for worker nodes is the `default` one. It allows all egress and ingress traffic by default on your private network.
+
+```bash
+openstack security group rule list default
++--------------------------------------+-------------+-----------+-----------+------------+-----------+-----------------------+----------------------+
+| ID                                   | IP Protocol | Ethertype | IP Range  | Port Range | Direction | Remote Security Group | Remote Address Group |
++--------------------------------------+-------------+-----------+-----------+------------+-----------+-----------------------+----------------------+
+| 0b31c652-b463-4be2-b7e9-9ebb25d619f8 | None        | IPv4      | 0.0.0.0/0 |            | egress    | None                  | None                 |
+| 25628717-0339-4caa-bd23-b07376383dba | None        | IPv6      | ::/0      |            | ingress   | None                  | None                 |
+| 4b0b0ed2-ed16-4834-a5be-828906ce4f06 | None        | IPv4      | 0.0.0.0/0 |            | ingress   | None                  | None                 |
+| 9ac372e3-6a9f-4015-83df-998eec33b790 | None        | IPv6      | ::/0      |            | egress    | None                  | None                 |
++--------------------------------------+-------------+-----------+-----------+------------+-----------+-----------------------+----------------------+
+```
+
+For now it is recommended to leave these security rules as-is or the nodes could be disconnected from the cluster.
+
+### Reserved IP ranges
+
+The following ranges are used by the cluster, and should not be used on the private network attached to the cluster.
+
+```text
+10.240.0.0/13 # Subnet used by pods
+10.3.0.0/16 # Subnet used by services
+```
+
+These ranges will be configurable in a future version.
+
+## Getting started
+
+The following methods are supported to create a MKS Premium cluster:
```

Comment:
```suggestion
The following methods are supported to create an MKS Premium cluster:
```

Reply:
I think that 'an' is the correct one as MKS (ɛm-keɪ-ɛs) is starting with a vowel sound